src/components/containers/NavSearch/NavSearch.js

```diff
@@ -1,29 +1,30 @@
 import React from 'react';
+import Dropdown from 'simple-react-dropdown'
+import SearchContent from '../SearchContent/SearchContent'
 
-const NavSearch = props => <div className="wrap-search wrap-header-search">
-  <div className="header-search">
-    <div className="control-group left control-search-select dropdown">
-      <button className="btn btn-dropdown-toggle btn-search-toggle"><i className="btn-icon fa fa-bars"/></button>
-      <div className="dropdown-menu dropdown-menu-search-select dropdown-menu-panel dropdown-menu-left dropdown-menu-small-size">
-        <div className="heading">Search Options</div>
-        <div className="dropdown-menu-wrap-list">
-          <div className="dropdown-menu-list">
-            <div className="dropdown-menu-item"><span className="dropdown-menu-item-text">Patient Search - Basic</span></div>
-            <div className="dropdown-menu-item"><span className="dropdown-menu-item-text">test</span></div>
+class NavSearch extends React.Component {
+  render () {
+    return(
+      <div className="wrap-search wrap-header-search">
+        <div className="header-search">
+          <div className="control-group left control-search-select dropdown">
+            <Dropdown content={<SearchContent />}>
+              <button className="btn btn-dropdown-toggle btn-search-toggle"><i className="btn-icon fa fa-bars"/></button>
+            </Dropdown>
+          </div>
+          <button className="btn btn-search"><i className="btn-icon fa fa-search"/></button>
+          <div className="wrap-search-holder">
+            <div className="search-holder">
```

Comment:
We should create separate component for it. Can we use <Input/> from react-bootstrap?